src/styles/App.scss

```
@@ -147,15 +163,53 @@ button {
   cursor: pointer;
 }
 
-@media screen and (max-width: 1080px) {
+@media screen and (max-width: 1210px) {
   .presentation {
-    padding: 20rem 0 0 34rem;
+    padding: 18rem 0 0 36rem;
+    
+    h1 {
+      font-size: 50px;
+  
+      p {
+        margin: 0;
+
+        margin: 20px 0 20px;
+  
+        .my-name {
+          font: 64px Verdana;
+          font-weight: 800;
+        }
+      }
+    }
+
+    .my-page {
+      font: 20px Verdana;
+      font-weight: 100;
+    }
   }
 }
 
 @media screen and (max-width: 1044px) {
   .presentation {
     padding: 14rem 0 0 34rem;
+
+    text-align: center;
+    
+    padding: 0;
+    padding-top: 130px;
```

Vc tá colocando o padding e retirando ele?? vc pode usar calc para somar dimensões diferentes no css.

```suggestion
    padding: calc(14rem + 130px) 0 0 34rem;

    text-align: center;
    
    padding: 0;
```

---

Ah ok, I need to review some things, because I was just setting new attributes that overwrite the others in the tests. Sorry about that.